flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -45,14 +45,20 @@ def bar_chart(
                 event_ends_before.timestamp() * 10**3,
             ]
         }
+    mark_type = "bar"
+    mark_interpolate = None
+    if sensor.event_resolution == timedelta(0) and sensor.has_attribute("interpolate"):
+        mark_type = "area"
+        mark_interpolate = sensor.get_attribute("interpolate")
```

Comment:
Maybe we should check that the value is defined in the supported interpolation methods by vega-lite. At least, I would check if `len(sensor.get_attribute("interpolate") > 0`  to avoid crashing on an empty string.

Reply:
I now implemented validation before saving to the database, which also constitutes a start for validation of other special attributes.

Funnily enough, the one unsupported string that it doesn't crash on was actually .. an empty string. :face_with_spiral_eyes: